### ISSUE TITLE
Deleting an org user succeeds with error

### ISSUE THREAD

**Issue by javitonino** [CartoDB/cartodb#11263]
As an org owner, trying to delete a user sometimes seems to fail with errors such as:
<img width="945" alt="screen shot 2017-01-16 at 15 58 56" src="https://cloud.githubusercontent.com/assets/2141690/21988052/2b685a60-dc06-11e6-9803-a453c7a8d7dd.png">
<img width="886" alt="screen shot 2017-01-16 at 16 07 51" src="https://cloud.githubusercontent.com/assets/2141690/21988068/3444b4e4-dc06-11e6-884f-d367436692e0.png"> 

The problem seem to be that the user is deleted first in central, which triggers a deletion in the box. Then the box proceeds to delete the user, which no longer exists.

This sometimes replicate and sometimes it doesn't, no idea what the exact criteria is. For example, using juanignaciosl-ded-01 org, seems to reliable replicate. More investigation is needed when addressing this one.

cc @juanignaciosl

**Comment by juanignaciosl** [comment]
Does this happen at organization owner administration page? First time that I see this!

**Comment by javitonino** [comment]
Yep, it's actually deletions started from box with Central enabled. I have been unable to replicate locally, but it fails in some deds (not all, even with the same branch deployed) 😕
